### ISSUE TITLE
Using with mutilple webpack exports

### ISSUE THREAD

**Issue by joshcomley** [frankwallis/WebpackAspnetMiddleware#15]
Hi there,

Thank you for this project.

If I try to use multiple exports from webpack - or even wrap a single export in an array - the app never loads:

    module.exports = function (env) {
        var clientBundleConfig = {
            ...
        };
        return [clientBundleConfig];
    }

I am trying to use this project to serve an Angular Universal project, so I need a server and a client config exported from webpack.

Any thoughts?

Much appreciated,
Josh

**Comment by frankwallis** [comment]
I don't think this is supported at the moment, I will need to look at the webpack handler and confirm/fix

**Comment by joshcomley** [comment]
Thanks for the reply, will keep my eye out for an update!
